### ISSUE TITLE
tvos player doesn't have object "window" defined

### ISSUE THREAD

**Issue by JaymanGitHub** [snowplow/snowplow-javascript-tracker#609]
Was trying to use snowplow tracker in tvos player events but got error with "Can't find variable: window"

**Comment by alexanderdean** [comment]
Hi @JaymanGitHub - this isn't an environment we've tested before. Can you provide (much) more detail on the intended usage and your setup?

**Comment by JaymanGitHub** [comment]
<script type="text/javascript">
;(function(p,l,o,w,i,n,g){if(!p[i]){p.GlobalSnowplowNamespace=p.GlobalSnowplowNamespace||[];
p.GlobalSnowplowNamespace.push(i);p[i]=function(){(p[i].q=p[i].q||[]).push(arguments)
};p[i].q=p[i].q||[];n=l.createElement(o);g=l.getElementsByTagName(o)[0];n.async=1;
n.src=w;g.parentNode.insertBefore(n,g)}}

**(window,document,"script","//d1fc8wv8zag5ca.cloudfront.net/2.6.2/sp.js","snowplow"));**

The bottom line parameter 'window' is not an object in tvos player

**Comment by alexanderdean** [comment]
Hmm, if it doesn't have `window` then it doesn't sound like tvOS Player is a viable web browser environment - this tracker is designed to work in web browsers, so there's nothing we can fix here.

**Comment by JaymanGitHub** [comment]
Ok then, thanks for you reply.
